willsCheatMenu/src/hacks/pets.ts

```diff
@@ -37,9 +37,23 @@ new Hack(category.pets, "Get All Epics").setClick(async () => {
 	_.player.kennel.petTeam.forEach((v: any) => {
 		if (v && (v as any).assignRandomSpells) (v as any).assignRandomSpells();
 	});
-	Toast.fire("Success!", "All epics have been added!", "success");
+	Toast.fire("Success!", "All legacy epics have been added!", "success");
 });
 
+// TODO: I need Aura's ID
+new Hack(category.pets, "Get All Mythical Epics").setClick(async () => {
+	const epics = _.gameData.pet.filter(x => [158, 166, 168].includes(x.ID));
+	epics.forEach(x => {
```

Comment:
please just do 
```js
[158, 166, 168].forEach(x => {
		_.player.kennel.addPet(JSON.stringify(x), VERY_LARGE_NUMBER, 26376, 100);
	});
	```
	
	there's no need to do it like this

Reply:
how?